### PR TITLE
qa/suites/upgrade/luminous-x

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

**Comment by yuriw** [comment]
@jdurgin ps take a look and I will give it a try

File: qa/suites/upgrade/luminous-x/parallel/0-cluster/start.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+meta:
+- desc: |
+   Run ceph on two nodes,
+   with a separate client 0,1,2 third node.
```
Comment (amitkumar50) [inline qa/suites/upgrade/luminous-x/parallel/0-cluster/start.yaml]: @yuriw `0,1,2 third` shouldn't be something '0th, 1st, 2nd and 3rd`

File: qa/suites/upgrade/luminous-x/parallel/0-cluster/start.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  ceph:
+    log-whitelist:
+    - scrub mismatch
+    - ScrubResult
+    - wrongly marked
+    - (POOL_APP_NOT_ENABLED)
+    - overall HEALTH_
+    conf:
+      global:
+        enable experimental unrecoverable data corrupting features: "*"
```
Comment (amitkumar50) [inline qa/suites/upgrade/luminous-x/parallel/0-cluster/start.yaml]: @yuriw I feel should be `corruption` instead of `corrupting`.
What your thoughts?

Comment (yuriw) [reply qa/suites/upgrade/luminous-x/parallel/0-cluster/start.yaml]: @amitkumar50 I actually don't know :(

**Comment by liewegas** [comment]
I have a parallel attempt in https://github.com/ceph/ceph/pull/17247

**Comment by liewegas** [comment]
Sorry, I was thinking the rados one.  Reopening!

**Comment by yuriw** [comment]
http://pulpito.ceph.com/yuriw-2017-08-29_17:01:34-upgrade:luminous-x:parallel-master-distro-basic-smithi/
see cause http://tracker.ceph.com/issues/21181
http://pulpito.ceph.com/yuriw-2017-08-30_16:32:58-upgrade:luminous-x:stress-split-master-distro-basic-smithi/
http://pulpito.ceph.com/yuriw-2017-08-30_20:04:30-upgrade:luminous-x:stress-split-erasure-code-master-distro-basic-smithi/

**Pull request merged**